saleor/graphql/checkout/mutations.py

```diff
@@ -376,7 +367,7 @@ def perform_mutation(cls, _root, info, checkout_id, lines, replace=False):
         if variants and quantities:
             for variant, quantity in zip(variants, quantities):
                 try:
-                    add_variant_to_checkout(
+                    checkout = add_variant_to_checkout(
```

Comment:
IMO, no need to return and assign checkout here. Checkout is passed to function as a reference, so all changes will be visible after the function will be finished.  Second thing, it is strange that the method responsible for adding variant to checkout returns checkout